test/unit/ngsiv2/examples/contextRequests/updateContextCompressTimestamp1.json

```diff
@@ -0,0 +1,10 @@
+{
+  "state": {
+    "type": "Boolean",
+    "value": "true"
+  },
+  "The Target Value": {
```

Comment:
If this is supossed to be an attribute name in NGSIv2, it is not a good candidate... Have a look to "Field syntax restrictions" in http://telefonicaid.github.io/fiware-orion/api/v2/latest/. It should not include whitespace, as:

> Allowed characters are the ones in the plain ASCII set, except the following ones: control characters, whitespace, `&`, `?`, `/` and `#`.

Maybe it works now in the unit test but it will fail for sure when tested in e2e integration, as CB will not accept an attribute name that way.

Suggestion: "TheTargetValue"

Reply:
Fixed in e9f6475e7ed413e8ac2e192187ffbd45399b5103